src/App.tsx

```diff
@@ -20,9 +19,13 @@ function App() {
       </div>
       <h1>Vite + React</h1>
       <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
+        <Button
+          variant="contained"
+          color="info"
```

Comment:
ideally, these attributes should be inside button component and prefixed based on our style guide.
But for now, this is okay as we dont have a guide yet

No change needed